.github/workflows/test.yml

```diff
@@ -60,6 +60,22 @@ jobs:
     - name: Run Pytest
       run: python -m pytest -s -v -n=auto test/
 
+  testmetal:
+     name: Metal Tests
+     runs-on: macos-13
+
+     steps:
+     - name: Checkout Code
+       uses: actions/checkout@v3
+     - name: Set up Python 3.10
+       uses: actions/setup-python@v4
+       with:
+         python-version: '3.10'
+     - name: Install Dependencies
+       run: pip install -e '.[metal,testing]'
+     - name: Run Pytest
+       run: python -m pytest -s -v -n=auto test/
```

Comment:
Since the metal dependencies are installed, it should use the metal backend by default
https://github.com/geohot/tinygrad/blob/e8a23d43311ad751bb23919657e76a0e7382358d/tinygrad/lazy.py#L320-L325